pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/redigerbar/InnhentingInformasjonFraBruker.kt

```diff
@@ -0,0 +1,671 @@
+                    text(
+                        Bokmal to "Bankopplysninger",
+                        Nynorsk to "Bankopplysningar",
+                        English to "Bank details",
+                    )
+                }
+                paragraph {
+                    text(
+                        Bokmal to "Vi har behov for bankopplysninger for å kunne utbetale pensjonen din.",
+                        Nynorsk to "Vi treng bankopplysningar for å kunne betale ut pensjonen din.",
+                        English to "We need your bank details in order to be able to pay you your pension.",
+                    )
+                }
+                paragraph {
+                    text(
+                        Bokmal to "I Din pensjon på $DIN_PENSJON_URL kan du registrere nødvendige bankopplysninger. Gå inn på Din Profil og velg «Bankkonto for utbetaling». Du kan logge inn med BankID, Buypass, MinID eller Commfides.",
+                        Nynorsk to "I Din pensjon på $DIN_PENSJON_URL kan du registrere nødvendige bankopplysningar. Gå inn på Din Profil og vel «Bankkonto for utbetaling». Du kan logge inn med BankID, Buypass, MinID eller Commfides.",
+                        English to "You can register the necessary bank details in the “Din pensjon” portal at $DIN_PENSJON_URL. Log in on your personal page “Din Profil” and select “Bank account for payment”. You can log in using BankID, Buypass, MinID or Commfides.",
+                    )
+                }
+            }
+
+            showIf(saksbehandlerValg.amerikanskSocialSecurityNumber) {
+                title1 {
+                    text(
+                        Bokmal to "Amerikansk social security-nummer",
+                        Nynorsk to "Amerikansk social security-nummer",
+                        English to "American social security number",
+                    )
+                }
+                paragraph {
+                    text(
+                        Bokmal to "Vi har behov for ditt amerikanske social security-nummer. (Det vil bli behandlet konfidensielt og er utelukkende for innhenting av informasjon fra amerikanske trygdemyndigheter om eventuell trygdetid i USA.)",
+                        Nynorsk to "Vi treng ditt amerikanske social security-nummer. (Det vil bli handsama løynleg, men er nødvendig for å hente inn informasjon frå amerikanske trygdemyndigheiter om eventuell trygdetid i USA.)",
+                        English to "We need your American social security number. (Will be treated confidentially and is solely to be used for the purpose of requesting information from US authorities about possible insurance coverage in the United States.)",
+                    )
+                }
+            }
+
+
+
+            showIf(saksbehandlerValg.registreringAvSivilstand) {
+                paragraph {
+                    textExpr(
+                        Bokmal to "Personnummer/ID for ".expr() + fritekst("ektefellen/partneren/samboeren") + " din",
```

Comment:
Slik jeg har tenkt på fritekst så er det en melding til saksbehandler og har derfor gått for samme språk i friteksten for alle språklag.

Reply:
Ser at du har litt blanding mellom et fritekst-språk og alle i malen her.

Reply:
Det stammer fra doksys malen hvor du har <Fritekst:x/y> og x/y. Jeg har erstattet <Fritekst:bla> med fritekst("bla")

Reply:
Skal vi parkere denne og gjøre det om til en oppgave på trello å finne ut av muliple choice tekster?